README.md

```diff
@@ -135,7 +135,7 @@ Do you have questions? Do not worry, we have prepared a complete [FAQ](https://g
 ## How to Engage, Contribute, and Give Feedback
 
 Some of the best ways to [contribute](./.github/CONTRIBUTING.md) are to try things out, file issues, join in design conversations,
-and make pull-requests. Proposals for changes specific to MAUI can be found [here for discussion](../../issues).
+and make pull-requests. Proposals for changes specific to MAUI can be found [here for discussion](https://github.com/dotnet/maui/issues).
```

Comment:
While the old link works fine in GitHub, it won't if the README is viewed locally (e.g in VSCode).